index.src.html

```diff
@@ -2089,12 +2089,31 @@ <h3 id="security-cross-origin-leakage">Cross-origin Credential Leakage</h3>
       MUST NOT offer credentials to an origin in response to
       {{CredentialsContainer/get()}} without user mediation if the credential's
       origin is not an exact match for the calling origin. That is,
-      {{Credential}} objects for <code>https://example.com</code> would not be
-      returned directly to <code>https://www.example.com</code>, but could be
+      {{Credential}} objects for <code>https://example.com/</code> would not be
+      returned directly to <code>https://www.example.com/</code>, but could be
       offered to the user via the chooser.
     </li>
+    <li>
+      MAY store credentials without user mediation when
+      {{CredentialsContainer/store()}} is called if the credential was
+      retrieved via a user mediated {{CredentialsContainer/get()}} from a
+      PSL-matched domain and new domain is still PSL-matching the original
+      domain for which the credentials were stored.
   </ol>
 
+  <div class="example">
+    A user stores a credential for <code>https://www.example.com/</code> on
+    their computer. This credential is synced via the browser's syncing
+    technologies to the user's phone. The user visits
```

Comment:
Nit: I'd suggest dropping the sync bit. It's irrelevant to your example.

"""
A user stores a credential after creating an account for `https://accounts.example.com/`. When they wish to sign into `https://example.com/` in the future, the user agent can offer that credential in response to {{CredentialsContainer/get()}} with user mediation, as described above. If the user chooses that credential, the website can call {{CredentialsContainer/store()}} to ensure that the credential is available to `https://example.com/` in the future. Since the two origins have the same <a>registerable domain</a>, and the user indicated that the credentials are valid for both origins, the user agent can store a credential for `https://example.com/` without further prompting.
"""